[NEWSTACK-105] PSO 6.2.0-rc1 release for NVMeoF-RoCE support (#177)
purestorage/pso-csi: CHANGELOG.md

# Changelog

## v6.2.0-rc1 (07/22/2021)

#### Features:

* FlashArray NVMeoF-RoCE support (NVMeoF using RDMA over Converged Ethernet)

> * A fresh PSO installation is recommended to use this release

Reviewer: ```suggestion
* A fresh PSO installation is recommended to use this release only if you want to try the NVMeoF-RoCE feature.
```

> * A fresh PSO installation is recommended to use this release only if you want to try the NVMeoF-RoCE feature.

Reviewer 2: I would flip this sentence order, and then upgrade it to a "must" (I don't think I'd trust moving a cluster from iSCSI or FC to NVMe in place).
```suggestion
* If you want to try the NVMeoF-RoCE feature, a fresh PSO installation is required.
```
